plugins/inventory/proxmox.py

```diff
@@ -234,14 +241,21 @@ def _get_agent_network_interfaces(self, node, vmid, vmtype):
                 )
             )['result']
 
+            if "error" in ifaces and "class" in ifaces["error"] and (ifaces["error"]["class"] in ["Unsupported", "CommandDisabled"]):
+                # This happens on Windows, even though qemu agent is running, the IP address
+                # cannot be fetched, as it's unsupported, also a command disabled can happen.
+                return result
+
             for iface in ifaces:
```

Comment:
```suggestion
            for iface in ifaces:
                result.append({
                    'name': iface['name'],
                    'mac-address': iface['hardware-address'],
                    'ip-addresses': [
                        "%s/%s" % (ip['ip-address'], ip['prefix']) for ip in iface['ip-addresses']
                    ] if 'ip-addresses' in iface else []
                })
            
```

Defaulting to `[]` should make working with the list of interfaces in Ansible/Jinja easier.

Reply:
@Ajpantuso yes just did those changes, and gonna push now running the sanity check to make sure it's not complaining about lint.

Reply:
Done.